Test/git-issues/git-issue-356-errors.dfy

```diff
@@ -0,0 +1,226 @@
+// RUN: %dafny /compile:0 "%s" > "%t"
+// RUN: %diff "%s.expect" "%t"
+
+
+// Check every possible conversion range error among int,char,bv,real,ORDINAL
+
+// int to char
+method Test1(n: int)
+{
+  var ch := n as char;
+}
+
+method Test1b(n: int)
+  requires n < 65536
+{
+  var ch := n as char;
+}
+
+method Test1c(n: int)
+  requires 0 <= n
+{
+  var ch := n as char;
+}
+
+method Test1OK(n: int)
+  requires 0 <= n < 65536
+{
+  var ch := n as char;
+}
+
+
+// int to ORDINAL
+method Test2(n: int) {
+  var o: ORDINAL := n as ORDINAL; // must be non-negative // NO ERROR
```

Comment:
I'm thinking this must be a stale comment. It says "NO ERROR", but the .expect file below shows that it does produce an error (as it should).